app/src/main/res/values/strings.xml

```diff
@@ -434,9 +434,9 @@
     <!-- Preference description for marketing data collection, parameter is the app name (e.g. Firefox) -->
     <string name="preferences_marketing_data_description">Shares data about what features you use in %1$s with Leanplum, our mobile marketing vendor.</string>
     <!-- Title for experiments preferences -->
-    <string name="preference_experiments">Experiments</string>
+    <string name="preference_experiments">Studies</string>
     <!-- Summary for experiments preferences -->
-    <string name="preference_experiments_summary">Allows Mozilla to install and collect data for experimental features</string>
+    <string name="preference_experiments_summary">Allows Mozilla to install and run studies</string>
```

Comment:
Copied from Desktop copy.

Reply:
We need to add a suffix `_2` since we are updating the strings so that they can be picked up and translated.